test/test-defs.cmake

```diff
@@ -100,11 +101,12 @@ if(PG_REGRESS)
     TEST_OUTPUT_DIR=${TEST_OUTPUT_DIR}
     TEST_SCHEDULE=${TEST_SCHEDULE}
     PG_BINDIR=${PG_BINDIR}
-    PG_REGRESS=${PG_REGRESS})
+  )
 endif()
 
 if(PG_ISOLATION_REGRESS)
   set(PG_ISOLATION_REGRESS_ENV
+    COMMAND=${PG_ISOLATION_REGRESS}
```

Comment:
I suggest keeping `PG_REGRESS` and just setting `PG_REGRESS=${PG_ISOLATION_REGRESS}` instead.

Also, `COMMAND` is a special keyword in CMake so might want to avoid.